src/main/java/com/paneedah/weaponlib/crafting/CraftingEntry.java

```diff
@@ -67,4 +96,11 @@ public String toString() {
 		}
 	}
 
+	public double getYield(){
+		if(this.item instanceof ManufacturingItemBase && this.yield == 1)
+			return ((ManufacturingItemBase)this.item).getRecoveryChance();
+		else
+			return this.yield;
```

Comment:
What happened to the formatting `ctrl` + `alt` + `L` is your friend

Reply:
didnt even know that was a thing 🤣